samples/general/mobile_nerf/mobile_nerf.h

```diff
@@ -85,26 +85,6 @@ class MobileNerf : public ApiVulkanSample
 		}
 	};
 
-	struct VertexHash
-	{
-		size_t operator()(Vertex const &vertex) const
-		{
-			return std::hash<glm::vec3>()(vertex.position) ^
-			       (std::hash<glm::vec2>()(vertex.tex_coord) << 1);
-		}
-	};
-
-	struct Texture_Input
```

Comment:
Replaced with the `Texture` class returned by the `ApiVulkanSample::load_texture` method.